rskj-core/src/main/java/co/rsk/core/bc/BlockExecutor.java

```diff
@@ -718,8 +702,7 @@ private BlockResult executeSequential(
 
             TransactionReceipt receipt = new TransactionReceipt();
             receipt.setGasUsed(gasUsed);
-            receipt.setCumulativeGas(totalGasUsedInBucket);
-
+            bucketGasAccumulated.ifPresent(receipt::setCumulativeGas);
```

Comment:
Well, actually, this's weird. Because it's always present, otherwise [this](https://github.com/rsksmart/rskj/blob/de130e96e0d59e8cd1e1665d894a902ecf7b5899/rskj-core/src/main/java/co/rsk/core/bc/BlockExecutor.java#L666) line is either returning or continuing the loop.

Reply:
What if `acceptInvalidTransactions` is true?

Reply:
I got it, you're right!